src/components/tables/DataTableCell.vue

```diff
@@ -42,6 +43,10 @@ export default {
   },
   methods: {
     transformDate: function(date) {
+      if (date === null) {
+        return '-';
```

Comment:
I'm using a — instead of a - here. It's a longer dash. If you're not sure, copy the character I've suggested.

```suggestion
        return '—';
```

Reply:
Fixed